### ISSUE TITLE
Want to reduce image size and reduce space. 

### ISSUE THREAD

**Issue by Vishal-Parasram** [Vishal-Parasram/Portfolio#11]
Issue regarding reducing the size of an image on mobile view and eliminating any unnecessary space at the bottom. The goal is to ensure that the image is properly sized for mobile devices and to optimize the layout for a better user experience. Also want to remove animation. Thank you.

**Comment by saimadhav24** [comment]
@Vishal-Parasram let me try it, if you are okay with it

**Comment by Vishal-Parasram** [comment]
@saimadhav24 Go ahead

**Comment by saimadhav24** [comment]
![image](https://user-images.githubusercontent.com/124759892/230722710-cbcb6f61-30ee-48a6-9e25-27e2c957ac4f.png)

<p>Is it fine like this?</p>

**Comment by Vishal-Parasram** [comment]
Yes it is looking good and also apply about me to flex wrap. Then create a PR.

**Comment by saimadhav24** [comment]
ok

**Comment by saimadhav24** [comment]
It was looking like this when i came to full screen mode
![image](https://user-images.githubusercontent.com/124759892/230724160-aa6415f3-6eeb-4f54-8dbd-0bb94b3258c5.png)

so i made it like this

![image](https://user-images.githubusercontent.com/124759892/230724200-5e9e6ca3-0d48-48a4-8564-3aafbb439bae.png)

if any queries let me know

**Comment by saimadhav24** [comment]
Final website:

![image](https://user-images.githubusercontent.com/124759892/230724623-a4a5359c-9b86-4ac8-a371-27652f5329c8.png)

![image](https://user-images.githubusercontent.com/124759892/230724640-ce51b0d7-b260-47d9-ae47-1562a22496a5.png)

![image](https://user-images.githubusercontent.com/124759892/230724652-3788445e-35d9-4116-98af-48e7759a6f27.png)

![image](https://user-images.githubusercontent.com/124759892/230724665-39498e33-7f22-4205-aa92-4565c4dc9e27.png)

![image](https://user-images.githubusercontent.com/124759892/230724693-f8e3cd41-68d8-4592-b8ff-1e07b08320db.png)

![image](https://user-images.githubusercontent.com/124759892/230724704-b9e45c56-f24a-480b-942f-94a5b2c50ce2.png)

![image](https://user-images.githubusercontent.com/124759892/230724719-9e1f1577-bb78-48f8-9c0a-f05b08a4ce4f.png)

![image](https://user-images.githubusercontent.com/124759892/230724731-82f2de6d-281e-41c8-b489-d7a7e473219e.png)

**Comment by saimadhav24** [comment]
@Vishal-Parasram Check the above and let me know

**Comment by Vishal-Parasram** [comment]
Good did you reduce image size for mobile view?

**Comment by saimadhav24** [comment]
yes, u can check it in screenshots

**Comment by Vishal-Parasram** [comment]
@saimadhav24 Create a PR request

**Comment by saimadhav24** [comment]
I have created it https://github.com/Vishal-Parasram/Portfolio/pull/12

**Comment by Vishal-Parasram** [comment]
Ok

**Comment by Vishal-Parasram** [comment]
@saimadhav24 hamburger icon is disappeared??

**Comment by saimadhav24** [comment]
Oh let me check

**Comment by Vishal-Parasram** [comment]
Ol@saimadhav24 ok

**Comment by saimadhav24** [comment]
can you share me the screenshot?

**Comment by Vishal-Parasram** [comment]
@saimadhav24 Appeared. Can you do one more thing reduce bottom gap between social media  icon so that email and icon are come to side by side.

**Comment by saimadhav24** [comment]
This is a new approach i m trying 
![image](https://user-images.githubusercontent.com/124759892/230741162-84a74df4-98f1-4691-a9bb-cb9821bf7c4f.png)

**Comment by saimadhav24** [comment]
![image](https://user-images.githubusercontent.com/124759892/230742722-7a475cbf-c98d-45d9-9b05-c884dfa359eb.png)

![image](https://user-images.githubusercontent.com/124759892/230742729-ec9b5cbd-37b9-43cc-ab67-8208477ee002.png)

![image](https://user-images.githubusercontent.com/124759892/230742731-b8e67a7c-4396-47c6-84c6-60628cf55bed.png)

![image](https://user-images.githubusercontent.com/124759892/230742740-9bd7f1d9-82a8-4b0c-8b69-32f52607c15e.png)

## Suggestion: it would be better if you remove the Service and About part in the bottom

![image](https://user-images.githubusercontent.com/124759892/230742778-42e22926-4af4-43b8-a9d5-dd468eaafa0b.png)
![image](https://user-images.githubusercontent.com/124759892/230742788-b690d5e0-14d1-451e-8f33-b30e2323e659.png)
![image](https://user-images.githubusercontent.com/124759892/230742878-a6abceaa-d319-4f77-8ec2-725cf3e10967.png)
![image](https://user-images.githubusercontent.com/124759892/230742883-7a1f5ea3-e5c4-40c7-93ca-a1b70f254ca2.png)
